### DISCUSSION TITLE
Website for this Project

### DISCUSSION THREAD

**Discussion by avinashkranjan** [avinashkranjan/Amazing-Python-Scripts#155] [Ideas]
# AIM

The minimum features I am looking for is:

- Static Landing Page which mentions the Core Objective of the Project
- A-List of Scripts in mentioned along with their description
- Contributors List

**Comment by Priyanshu-official** [comment]
@avinashkranjan Sorry for late I am not well but Now I am back 
I want to work on one project in a proper manner so I choose this Project
I am Participant SWOC 2021

**Comment by Priyanshu-official** [comment]
@avinashkranjan  We can start with the design process if you have any design in your mind let me know?
In this, we use the React component so we can divide the section and Everyone get a chance to contribute...
also for **Framework** from my side, we can use [Tailwindcss](https://tailwindcss.com/)
Pls, give your opinion...

**Comment by avinashkranjan** [comment]
@Priyanshu-official I'm confused between React/HTML+CSS+JS... Since HTML+CSS+JS we can get more contributors and work would be fast... But React on the other would be great option too.. I'm waiting for other participants to reply... So I can know their Tech Stack... Of they don't reply we can go get started with React + Tailwind...

**Reply by fatmab2809** [reply]
I don't know React, I can contribute using HTML, CSS & JS using Bootstrap framework. But I am okay with learning a new framework to make a contribution.

**Reply by avinashkranjan** [reply]
Great Then, We can work with ReactJS + Tailwind then..

**Reply by Priyanshu-official** [reply]
@avinashkranjan Pls Let me know when we start.
Till then we can see how may Contributor are interested

**Reply by Priyanshu-official** [reply]
Good Combination 
HTML CSS JS + FRAMEWORK
we can more contributor I am Agree With this... @avinashkranjan

**Comment by vaishnavirshah** [comment]
Hi, I  am willingly to work on this issue but I am not familiar with React, is there a chance we can do it using bootstrap?

**Comment by avinashkranjan** [comment]
I See...So We should get started with website Using HTML, CSS, JS..
So We First We need a 

- Static Landing Page which mentions the Core Objective of the Project
- A-List of Scripts in mentioned along with their description
- Contributors List

So, The Work would be divided accordingly.... You can Input More Idea and after approval start working on it.. All the changes should be pushed on `gh-pages` branch..You can comment on what you'll be doing and get started.. You can work alone or 2-3 peoples on a page..Create an Issue and get started..

**Reply by vaishnavirshah** [reply]
Can I get started with the landing page ? I will make a simple layout with bootstrap and then other things can be added.

**Reply by fatmab2809** [reply]
After the layout is made, I can add the list of scripts. Each script can be in the form of a card. For this, if anyone wants to work together, it'll be done faster.

**Reply by avinashkranjan** [reply]
Sure You guys can get started..Lemme knw if anything is needed..Create an Issue I'll Assign it you for documentation purposes..

**Reply by jainstuti** [reply]
> After the layout is made, I can add the list of scripts. Each script can be in the form of a card. For this, if anyone wants to work together, it'll be done faster.

I can work with you for this

**Reply by avinashkranjan** [reply]
Hey @jainstuti You can check out the the one which is already made on `gh-pages` branch and suggest your improvements

**Comment by NChechulin** [comment]
Maybe we should consider using some static generator like [Hugo](https://github.com/gohugoio/hugo)? The building process could be automated with GH Actions and the website could be hosted on GH Pages.
I believe the only piece of effort would be writing a python/bash script that takes project names and copies README files from each project.

**Reply by avinashkranjan** [reply]
That Sounds like a great idea too..

**Comment by SHITIZ-AGGARWAL** [comment]
I made a rough ui design for scripts page , many other features like tags, filters etc can be included . 

https://www.figma.com/proto/2JjiNsWQz8u5z63QwDGsrv/Untitled?node-id=1%3A2&viewport=281%2C360%2C1.124820590019226&scaling=min-zoom

**Reply by Priyanshu-official** [reply]
Recent Script design is awesome 👍💯

**Comment by Priyanshu-official** [comment]
@avinashkranjan sorry Bit late Exam Is going on 
And O don't know why I am not getting Notified 😅
Well Website Work is started that's great Hope I catch up all of you on time.
I start Contributing in few days 🤗 you guys can Have fun And code

**Comment by avinashkranjan** [comment]
Have Created an Issue for Creating Contributors Section. If Anyone Interested Go ahead

**Comment by SurbhiRathi23** [comment]
@avinashkranjan the website in gh-pages has some issues
- navbar is not clearly visible its color could be changed
- search bar have no use in home page, we should only keep search bar in "scripts page"
- also contributors list doesn't show correct names, it showing A for everyone

I can work on these issues. I am a GSSOC participant so you can assign this part to me as a part of gssoc21.

![6](https://user-images.githubusercontent.com/55491101/110514130-ff7f2980-812c-11eb-920e-ce65c3946337.JPG)
![7](https://user-images.githubusercontent.com/55491101/110514141-01e18380-812d-11eb-800c-eaaf8945e102.JPG)

**Reply by avinashkranjan** [reply]
Create an Issue and You can start working on it..

**Reply by SurbhiRathi23** [reply]
okay
